### DISCUSSION TITLE
question about build process

### DISCUSSION THREAD

**Discussion by calvin2021y** [sourcelair/xterm.js#4415] [Q&A]
I want build a minimal dist file with `FitAddon.js` and others support.  I clone the project and run npm run build. 

The output from file `addons/xterm-addon-fit/out/FitAddon.js` look like:

```js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.FitAddon = void 0;
const MINIMUM_COLS = 2;
const MINIMUM_ROWS = 1;
class FitAddon {
````

Is there a way to made old fashion dist file for FitAddon?

**Comment by jerch** [comment]
The addon folders contain a webpack config, that will bundle things into a single distributable.

**Reply by Tyriar** [reply]
Also you can see exactly how we package and release by looking at the azure pipelines yml https://github.com/xtermjs/xterm.js/blob/9154831afa9aea78306d8edad8fe403616758ad3/azure-pipelines.yml#L144-L176
